include/tvm/ir_operator.h

```diff
@@ -528,6 +530,36 @@ inline Expr MakeConstScalar(Type t, ValueType value) {
   if (t.is_int()) return ir::IntImm::make(t, static_cast<int64_t>(value));
   if (t.is_uint()) return ir::UIntImm::make(t, static_cast<uint64_t>(value));
   if (t.is_float()) return ir::FloatImm::make(t, static_cast<double>(value));
+  if (DatatypeRegistry::Global()->DatatypeRegistered(
```

Comment:
Call in to datatype registry -- need something that can take the requested type and return a const scalar of that type that is equivalent in value to our custom datatype

Reply:
@tqchen can you check over how I implemented this?